string connected_to
string through_door

# The data structure to represent a 2D coordinate

float32 x   
float32 y   
